### PR TITLE
Add guided bombs to F-15E Suite 4+ loadouts

### PR DESCRIPTION
**Pull request by Starfire13**
GBUs have been added to the F-15E Suite 4+ loadouts

Also switched CAP loadouts to 2 external tanks instead of 3 to improve aircraft performance, as 2 tanks is plenty for players and the AI has infinite fuel now.

### CODE REVIEW

File: resources/customized_payloads/F-15ESE.lua [hunk 1]
```diff
@@ -2,170 +2,158 @@ local unitPayloads = {
 	["name"] = "F-15ESE",
 	["payloads"] = {
 		[1] = {
-			["displayName"] = "Liberation Strike",
-			["name"] = "Liberation Strike",
+			["displayName"] = "AIM-9M x 4, AIM-120C x 2, CBU-87 x 6, TGP",
```
Comment (DanAlbert) [inline resources/customized_payloads/F-15ESE.lua]: This does nothing in Liberation.

**Comment by DanAlbert** [comment]
Since I was slow at getting to this, you may want to update it with JDAM now?

**Comment by Starfire13** [comment]
Ok, something's wrong. I already updated the loadouts for JDAMs. But they don't appear to be there. And there's all these other loadouts in the file that I didn't create. Let me doublecheck.

File: resources/customized_payloads/F-15ESE.lua (hunk 1 above)
Comment (Starfire13) [reply resources/customized_payloads/F-15ESE.lua]: Yeah, I dunno why this got added in to the file. This is one of the default F-15E loadouts, I believe. I'll fix it.

**Comment by DanAlbert** [comment]
Oh, if JDAM is already there then ignore me. I just assumed they weren't because liberation only learned about those a few minutes ago :)

**Comment by Starfire13** [comment]
> Oh, if JDAM is already there then ignore me. I just assumed they weren't because liberation only learned about those a few minutes ago :)

Yeah, I added them in preparation for the update. :)

**Pull request merged**